oap-server/server-alarm-plugin/src/main/java/org/apache/skywalking/oap/server/core/alarm/provider/AlarmRulesWatcher.java

```diff
@@ -46,16 +48,24 @@
     private volatile Map<AlarmRule, RunningRule> alarmRuleRunningRuleMap;
     private volatile Rules rules;
     private volatile String settingsString;
+    @Getter
+    private volatile CompositeRuleEvaluator compositeRuleEvaluator;
```

Comment:
`compositeRuleEvaluator` is effectively `final`, and doesn't need `volatile`

Reply:
done